lib/docs/stories/introduction/usage.stories.mdx

```diff
@@ -1,42 +1,63 @@
 import { Meta } from "@storybook/addon-docs/blocks";
 
-<Meta title="Introduction/Usage" />
+<Meta title="Introduction/Getting Started" />
 
-# Usage
+# Getting Started
 
-Set up and use PubNub Chat Components for React to build your own chat application.
+Create a sample chat app containing:
+
+- [**Message input**](https://pubnub.github.io/react-chat-components/docs/?path=/docs/components-message-input--default) field
+- [**Message list**](https://pubnub.github.io/react-chat-components/docs/?path=/docs/components-member-list--default) that will show when you start typing and sending messages
 
 ## Requirements
 
-- React v16.8+ and ReactDOM v16.8+
+You will use the following tools in the course of this guide:
+
+- [React v16.8+](https://www.npmjs.com/package/react/v/16.8.0) and [ReactDOM v16.8+](https://www.npmjs.com/package/react-dom)
 - [PubNub JavaScript SDK v4.29+](https://www.pubnub.com/docs/sdks/javascript/)
 - [PubNub React SDK v2.1.0+](https://www.pubnub.com/docs/chat/react/setup)
+- Code editor of your choice (e.g. [Visual Studio Code](https://code.visualstudio.com/download))
+
+## Steps
+
+Let's get started!
+
+### Create a PubNub account
+
+Before you start, you need to get **Publish Key** and **Publish Key** for your chat app.
+Sign in or create an account on the [Admin Portal](https://dashboard.pubnub.com/) to create an app and a new keyset. 
 
-## Set up PubNub account
+>**TIP**: When you create a new app on the Admin Portal, the first set of keys is generated automatically, but a single app can have as many keysets as you like. We recommend that you create separate keysets for production and test environments. There are some of the functionalities you might need to enable on your keyset depending on the use case, but they won't be required to complete this Getting Started guide. These options include:
```

Comment:
From what I tested, when you create a new PubNub account and select "Chat" as your use case, most of those options will already be enabled. Perhaps we should mention that here?